src/manager.cpp

```diff
@@ -437,6 +437,9 @@ sdbusplus::async::task<bool>
         case 14: // Error in IPC code
         case 22: // Error allocating core memory buffers
         {
+            co_await _extDataIfaces->createErrorLog(
+                "xyz.openbmc_project.RBMC_DataSync.Error.SyncFailure",
+                ext_data::ErrorLevel::Error, {});
```

Comment:
No procedure callout? we discussed to callout with the procedure BMC0002, right?